src/storage-operators/src/s3_oneshot_sink.rs

```diff
@@ -84,7 +85,7 @@ pub fn copy_to<G, F>(
         }
 
         let sdk_config = match aws_connection
-            .load_sdk_config(&connection_context, connection_id)
+            .load_sdk_config(&connection_context, connection_id, InTask::No)
```

Comment:
```suggestion
            .load_sdk_config(&connection_context, connection_id, InTask::Yes)
```
We want this to be run in a tokio task

Reply:
omg rip, thanks for the look!

Reply:
my brain was frying